aspnetcore/includes/spa-templates-new-vs-old.md

```diff
@@ -0,0 +1,25 @@
+---
+author: tdykstra
+ms.author: tdykstra
+ms.date: 09/12/2023
+ms.topic: include
+---
+## Newer vs. older SPA templates
+
+The SPA templates that use the special project type for JavaScript and TypeScript are available in Visual Studio 2022 version 17.7 or later with the **ASP.NET and web development** workload installed. They're available only in Visual Studio, not by using the `dotnet new` command of the .NET CLI.
```

Comment:
The new SPA templates that include an ASP.NET Core backend will ship their first stable release with 17.8. Also, let's not draw unnecessary attention to the lack of CLI support. We think the existing SPA templates saw very little usage from the CLI. If it turns out users want CLI support for project creation, we'll consider adding it.

 ```suggestion
The SPA templates that use the special project type for JavaScript and TypeScript are available in Visual Studio 2022 version 17.8 or later with the **ASP.NET and web development** workload installed.
```